charts/topolvm/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+type: application
+
+home: https://github.com/topolvm/topolvm
+name: topolvm
+description: Topolvm
+version: 1.3.0
+appVersion: 0.8.3
+kubeVersion: ">=1.18.0"
```

Comment:
@d-kuro please check this.

Reply:
Currently, Kubernetes v1.21 is not supported, so does that mean I should do the following?

```
kubeVersion: ">=1.18.0 < 1.21.0"
```
https://github.com/topolvm/topolvm/blob/main/README.md

I think the restriction should be on the min version only, as this would prevent all users of Kubernetes v1.21 and above from using the Helm chart.
What do you think?

```console
$ helm install --create-namespace --namespace=topolvm-system topolvm ./charts/topolvm --dry-run
Error: chart requires kubeVersion: >=1.18.0 < 1.21.0 which is incompatible with Kubernetes v1.21.1
```

Reply:
You are right. It is sufficient that specifying the minimum version.